app/views/users/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col-md-4">
+    <h1>Welcome <%= @user.name %> </h1>
+  </div>
+  <div class="col-md-8">
+    <% if @user.events.any? %>
+      <h2>Events</h2>
+      <ol>
+        <% @user.events.each do |event| %>
+          <li>
+            <span>Event: <%= event.desc %></span><br>
+            <span>Date: <%= event.date %></span><br>
+          </li>
+        <% end %>
```

Comment:
Why `@users.events` when you have created a variable `@events` in the action `users#show`